initrd/bin/oem-factory-reset

```diff
@@ -354,6 +378,22 @@ report_integrity_measurements()
     fi
 }
 
+usb_security_token_capabilities_check()
+{
+    TRACE "Under /bin/oem-factory-reset:usb_security_token_capabilities_check"
+
+    enable_usb
+    # ... first set board config preference
+    if [ -n "$CONFIG_GPG_ALGO" ]; then
+      GPG_ALGO=$CONFIG_GPG_ALGO
+			DEBUG "Setting \$GPG_ALGO to (board-)configured: $CONFIG_GPG_ALGO"
```

Comment:
No need to escape $GPG_ALGO here.
Also, to display board info to DEBUG it would be $CONFIG_BOARD not (board)-configured.

We would want something like: 
DEBUG "$CONFIG_BOARD overridden global default defined GPG_ALGO  $GPG_ALGO to $CONFIG_GPG_ALGO"

Note that this detail will be hidden to the end user unless in debug mode.
@JonathonHall-Purism thoughts? Should those be standard echo or DEBUG info?